automation/repeated_test.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+SCRIPT_PATH="$(
+    cd "$(dirname "${BASH_SOURCE[0]}")/" || exit
+    echo "$(pwd)/"
+)"
+
+NUM_TESTS=${NUM_TESTS:-3}
+if [[ -z ${NEW_TESTS} ]]; then
+    NEW_TESTS=$("${SCRIPT_PATH}"/new_tests.sh)
+    # skip certain tests for now, as we don't have a strategy currently
+    NEW_TESTS=$(echo "$NEW_TESTS" | sed -E 's/\|?[^\|]*(sriov|multus|windows|genie|gpu)[^\|]*//g')
+fi
+if [[ -z "${NEW_TESTS}" ]]; then
+    echo "Nothing to test"
+    exit 0
+fi
+
+TEST_LANES=( 'k8s-1.14.6' 'k8s-1.15.1' 'k8s-1.16.2' )
+
+trap '{ make cluster-down; }' EXIT SIGINT SIGTERM
+
+for lane in "${TEST_LANES[@]}"; do
+
+    echo "test lane: $lane, preparing cluster up"
+
+    export KUBEVIRT_PROVIDER="$lane"
+    export KUBEVIRT_NUM_NODES=2
+    make cluster-up
+
+    for i in $(seq 1 "$NUM_TESTS"); do
+        echo "test lane: $lane, run: $i"
+        make cluster-sync
+        ginko_params="--ginkgo.noColor --ginkgo.focus=${NEW_TESTS} --ginkgo.regexScansFilePath=true"
+        FUNC_TEST_ARGS="$ginko_params" make functest
+        if [[ $? -ne 0 ]]; then
+            echo "test lane: $lane, run: $i, tests failed!"
+            exit 1
+        fi
+    done
+
+    make cluster-down
```

Comment:
Since you have a trap for `EXIT`, this line here is redundant and will actually call `make cluster-down` twice

Reply:
Yes, but calling `cluster-down` twice should not be a problem, right? But if we don't call `cluster-down` here the loop would not work any more, as the next cluster-up (line 29) wouldn't work any more.

Reply:
you are right, I forgot the loop